app/ide-desktop/lib/content/src/index.ts

```diff
@@ -188,44 +81,51 @@ class Config {
 
 class Main {
     async main(inputConfig: any) {
-        const config = Object.assign(
-            {
-                pkgWasmUrl: 'assets/pkg-opt.wasm',
-                pkgJsUrl: 'assets/pkg.js',
+        // FIXME: use inputConfig
+        const config = {
+            loader: {
+                wasmUrl: 'assets/pkg-opt.wasm',
+                jsUrl: 'assets/pkg.js',
                 shadersUrl: 'assets/shaders',
             },
-            inputConfig
-        )
+        }
+
         const appInstance = new app.App({
             config,
-            configExtension: new Config(),
+            configOptions: options,
             packageInfo: {
+                // @ts-ignore
                 version: BUILD_INFO.default.version,
+                // @ts-ignore
                 engineVersion: BUILD_INFO.default.engineVersion,
             },
         })
 
         if (appInstance.initialized) {
-            if (appInstance.config.params.dataGathering.value) {
+            if (options.options.dataCollection.value) {
                 // TODO: Add remote-logging here.
```

Comment:
todo, an old one

Reply:
It is TODO for the cloud team. Leaving it here. They are working on it RN.